azure-stack/hci/manage/manage-syslog-forwarding.md

```diff
@@ -0,0 +1,469 @@
+
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on>
+```
+
+In case you want to test the integration of your syslog server with the Azure Stack HCI syslog forwarder by using a self-signed or untrusted certificate, you can use these flags to skip the server validation done by the client during the initial handshake.
+
+1. Skip validation of the Common Name value in the server certificate. Use this flag if you provide an IP address for your syslog server.
+
+   ```azurepowershell
+   Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> 
+   -SkipServerCNCheck
+   ```
+
+1. Skip entirely the server certificate validation.
+
+   ```azurepowershell
+   Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on>  
+   -SkipServerCertificateCheck
+   ```
+
+> [!IMPORTANT]
+> Microsoft recommends that you do not use the `-SkipServerCertificateCheck` flag in production environments.
+
+### Syslog forwarding with TCP and no encryption
+
+In this configuration, the syslog client in Azure Stack HCI forwards messages to the syslog server over TCP, with no encryption. The client doesn’t verify the identity of the server nor does it provide its own identity to the server for verification.
+
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -NoEncryption
+```
+
+> [!IMPORTANT]
+> Microsoft recommends that you do not use this configuration in production environments.
+
+### Syslog forwarding with UDP and no encryption
+
+In this configuration, the syslog client in Azure Stack HCI forwards messages to the syslog server over UDP, with no encryption. The client doesn’t verify the identity of the server nor does it provide its own identity to the server for verification.
+
+```azurepowershell
+Set-AzSSyslogForwarder -ServerName <FQDN or ip address of syslog server> -ServerPort <Port number on which the syslog server is listening on> -UseUDP
+```
+
+While UDP with no encryption is the easiest to configure, it doesn’t provide any protection against man-in-the-middle attacks or eavesdropping of messages.
+
+> [!IMPORTANT]
+> Microsoft recommends that you do not use this configuration in production environments.
+
+## Remove syslog forwarding
+
+Run the following command to remove the syslog forwarder configuration and stop the syslog forwarder:
+
+```azurepowershell
+Set-AzSSyslogForwarder -Remove 
+```
+
+## Verify syslog setup
+
```

Comment:
Verify syslog setup should be before and the remove syslog forwarding probably after.

Reply:
Done > revised to present logical sequence of sections/tabs.